### ISSUE TITLE
Hacktoberfest - Hyderabad, Bangalore and Mumbai

### ISSUE THREAD

**Issue by ranjithrajv** [mozilla/OpenDesign#220]
## Goal: 
Poster for Hacktoberfest [City]

## Info: 
Promoting in-person hacktoberfest gatherings in at any place

## Style Information: 
* Hacktoberfest and [City] fontsizes should be same
* Mozilla logo on top of them
* Highlight Venue and Date+Time 
* Section for Sponsors and Community Partners

## Deadline: 
20 Oct 2018

## Tag: 
[Design Needed]

**Comment by kindavishal** [comment]
Hi @ranjithrajv I guess we don't need this anymore. Would you like me to close this issue?

**Comment by ranjithrajv** [comment]
It happens this year too, so still need it.

**Comment by pransh15** [comment]
Can you please file a new one with the same details(more details related to each venues for each city + sponsors), please?

Since anyone who's available can view a newer issue with a more recent deadline.

**Comment by ranjithrajv** [comment]
We need template (a .svg file), so one can download, add details and publish.

**Comment by ruchikabgosain** [comment]
Hello @ranjithrajv and @kindavishal I'll be happy to take this up. Could you provide a sample poster or a previous one for any details? Or a website for this event? Let me know.

**Comment by kindavishal** [comment]
Great @ruchikabgosain :blush: I'll be around for this! The 2019 version doesn't have any updates yet. @ranjithrajv what did you have in mind?

**Comment by pransh15** [comment]
Yes, the updated graphics for Hacktoberfest will be released between late-August to September, I think.
Till then, we don't really have the official graphics to move forward with this.

I think we should close this and can open a new issue when Hacktoberfest 2019 is announced.

**Comment by kindavishal** [comment]
Well the goal was to create ```Template Poster for Hacktoberfest [City]``` I think he wanted a general template maybe?

**Comment by pransh15** [comment]
The General Template will include the Hacktoberfest logo and the new color scheme.
Without that, we don't really have anything to work on - apart from the city and venue.

Which is why:
> I think we should close this and can open a new issue when Hacktoberfest 2019 is announced.
